kubernetes/kubernetes.go

```diff
@@ -198,12 +198,13 @@ func (ka KubeAgentConfig) collectMetrics(
 	}
 
 	if config.RetrieveNodeSummaries {
-
 		err = retrieveNodeSummaries(config, msd, metricSampleDir, nodeSource)
 		if err != nil {
 			log.Printf("Warning: %s", err)
 		}
-	} else {
+	}
+
+	if config.CollectHeapsterExport {
```

Comment:
What's the default for this?

Reply:
https://github.com/cloudability/metrics-agent/blob/master/cmd/kubernetesCmd.go#L126